File: docs/text.rb
require "twilio-ruby"
# require "takeaway"

class Text
  #   def initialize
  #     # account_sid = "my account sid"
  #     # auth_token = "my auth token"
  #     # @client = Twilio::REST::Client.new(account_sid, auth_token)
  #     # @from = "+15005550006"
  #     # @to = "+chosen number"
  #     time = Time.new
  #   end

  def message
    # @client.messages.create(
    #   from: @from,
    #   to: @to,
    #   body: "Thank you! Your order was placed and will be delivered before #{time.hour + 1}:#{time.min}",
    # )
    return true
  end
end

File: takeaway-challenge-python/bm_takeaway_makers/text.py
from twilio.rest import Client
import os

class Text():

    def __init__(self, amount):
        self.amount = amount
    
    def send_text(self):
        ACCOUNT_SID = os.environ['TWILIO_SID']
        AUTH_TOKEN = os.environ['TWILIO_AUTH']
        client = Client(ACCOUNT_SID, AUTH_TOKEN) 
        body = "Amount to pay: " + self.amount
        message = client.messages.create(to = os.environ['PHONE'], 
            from_ = os.environ['TWILIO'],
            body = body
        )    